lib/dialects/mysql.ts

```diff
@@ -305,19 +305,15 @@ export default class MySQL implements SchemaInspector {
         rc.CONSTRAINT_NAME = kcu.CONSTRAINT_NAME
         AND kcu.CONSTRAINT_SCHEMA = rc.CONSTRAINT_SCHEMA
       WHERE
-        rc.CONSTRAINT_SCHEMA = ?;
+        rc.CONSTRAINT_SCHEMA = ?
+        ${table ? ` AND rc.TABLE_NAME = '${table}'` : ''}
```

Comment:
We should use the parameterized version `?` here to prevent a SQL injection attack

Reply:
Is this better? More inline with the rest of the code in the file